src/Scene.cpp

```
@@ -0,0 +1,74 @@
+#include "Scene.hpp"
+#include "Ray.hpp"
+#include <fstream>
+#include <iostream>
+#include <sstream>
+
+Raytracer::Scene::Scene() {
+  _path = "./output.ppm";
```

Cppcheck (2.10-2)

[performance] Variable '_path' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)